### PR TITLE
New design of profile to mobile devices

### PR DESCRIPTION
**Pull request by tiagolimpe**
**Feature/Bug description:** 
selection_025

**Solution:** ...

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/user/profile_mobile.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<md-dialog id="user-profile-size">
+    <div class="user-profile-dialog">
+        <div class="user-profile-cover">
+            <div md-colors="{background: 'default-teal-600'}">
+                <img>
```
Comment (JuliePessoa) [inline frontend/user/profile_mobile.html]: is it necessary?

File: frontend/user/profile_mobile.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            </div>
+            <div id="user-profile-img">
+                <img ng-src="{{profileCtrl.user.photo_url}}" />
+            </div>
+        </div>
+        <div class="user-profile-content">
```
Comment (JuliePessoa) [inline frontend/user/profile_mobile.html]: two divs. one inside the other. with no sibling content. couldnt just be one div?

File: frontend/user/profileController.js [hunk 3]
```diff
@@ -3,14 +3,17 @@
 (function () {
     const app = angular.module("app");
 
-    app.controller("ProfileController", function ProfileController(user, currentUserKey, UserService, $state, $mdDialog, STATES) {
+    app.controller("ProfileController", function ProfileController(user, currentUserKey, institutionKey, UserService, $state, $mdDialog, STATES) {
```
Comment (JuliePessoa) [inline frontend/user/profileController.js]: proper dependency annotation

File: frontend/user/profileService.js [hunk 4]
```diff
@@ -8,14 +8,17 @@
     app.service("ProfileService", function UserService($mdDialog, HttpService, $q, AuthService) {
         var service = this;
 
-        service.showProfile  = function showProfile(userKey, ev) {
+        service.showProfile  = function showProfile(userKey, ev, institutionKey) {
              $mdDialog.show({
-                templateUrl: 'app/user/profile.html',
+                parent: angular.element(document.body),
```
Comment (JuliePessoa) [inline frontend/user/profileService.js]: is it necessary?

File: frontend/user/profile_mobile.html [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+                <p class="user-profile-subtitle">{{ profileCtrl.showProperty(profileCtrl.currentProfile.email) }}</p>
+            </div>
+        </div>
+        <div layout="row" class="user-profile-action">
+            <div md-colors="{color: 'default-grey-600'}"
+                id="user-profile-show-div" ng-click="profileCtrl.closeDialog()">
+                <span class="user-profile-title">FECHAR</span>
+            </div>
+            <div md-colors="{color: 'default-teal-500'}"
+                id="user-profile-show-div" ng-click="profileCtrl.goToUserProfilePage()">
```
Comment (brobsc) [inline frontend/user/profile_mobile.html]: There's no function named profileCtrl.goToUserProfilePage()

File: frontend/user/profile_mobile.html [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            <div id="user-profile-info">
+                <p class="user-profile-title user-profile-default-color"><b>CARGO</b></p>
+                <p class="user-profile-subtitle">{{ profileCtrl.showProperty(profileCtrl.currentProfile.office) }}</p>
+                <p class="user-profile-title user-profile-default-color"><b>TELEFONE</b></p>
+                <p class="user-profile-subtitle">{{ profileCtrl.showProperty(profileCtrl.currentProfile.phone) }}</p>
+                <p class="user-profile-title user-profile-default-color"><b>E-MAIL</b></p>
+                <p class="user-profile-subtitle">{{ profileCtrl.showProperty(profileCtrl.currentProfile.email) }}</p>
+            </div>
+        </div>
+        <div layout="row" class="user-profile-action">
```
Comment (brobsc) [inline frontend/user/profile_mobile.html]: Shouldn't these be button elements?

**Pull request merged**